arduino/builder/sketch.go

```diff
@@ -265,7 +276,20 @@ func SketchCopyAdditionalFiles(sketch *sketch.Sketch, destPath string) error {
 			return errors.Wrap(err, "unable to create the folder containing the item")
 		}
 
-		err = writeIfDifferent(item.Path, targetPath)
+		var sourceBytes []byte
+		if override, ok := overrides[relpath]; ok {
+			// use override source
+			sourceBytes = []byte(override)
+		} else {
+			// read the source file
+			s, err := item.GetSourceBytes()
+			if err != nil {
+				return errors.Wrap(err, "unable to read contents of the source item")
+			}
+			sourceBytes = s
```

Comment:
it's a go language "glitch", if you try to set as:

`sourceBytes, err := item.GetSourceBytes()`

golang will create a new variable `sourceBytes` that shadows the external one, instead:

`sourceBytes, err = item.GetSourceBytes()`

will fail because err is not defined...

Reply:
Right, I'd have put `var err error` before but this works anyway so it's ok for me. Approving.